docs/src/snap/howto/cis-hardening.md

```diff
@@ -0,0 +1,270 @@
+# CIS compliance 
+
+CIS Hardening refers to the process of implementing security configurations that
+align with the benchmarks set by the [Center for Internet Security (CIS)][]. The
+open source tool [Kube-bench][] is designed to automatically check whether
+your Kubernetes clusters are configured according to the 
+[CIS Kubernetes Benchmark][]. This guide covers how to:
+
+- [Install and configure kube-bench](#install-kube-bench)
+- [Harden your deployments](#harden-your-deployments)
+- [Audit your deployments](#audit-your-deployments)
+
+## What you'll need
+
+- A bootstrapped {{product}} cluster (see the [Getting Started]
+[getting-started-guide] guide)
+- Root or sudo access to the machine
```

Comment:
I wonder if there is a different way of expressing this which doesn't start with 'Root' or 'Sudo'. The capital R is jarring for me

Reply:
Reworked to "you have root ..."